### PR TITLE
Custom sort for event listing

### PR DESCRIPTION
**Pull request by garyhtou**
#1156

I'm kinda suck

I changed `Event.reorder(:name)` to `Event.reorder(CUSTOM_SORT)` in the new disbursement view to test, but it seems to not work.

### CODE REVIEW

File: app/models/event.rb [hunk 1]
```diff
@@ -144,6 +144,12 @@ class Event < ApplicationRecord
 
   before_create :default_values
 
+  CUSTOM_SORT = "CASE WHEN id = 183 THEN '1'
+                      WHEN id = 999 THEN '2'
+                      WHEN id = 689 THEN '3'
+                      WHEN id = 636 THEN '4'
+                      ELSE name END ASC"
```
Comment (motdotla) [inline app/models/event.rb]: looks almost there from testing the raw sql. but needs a way to still put event names that start with a number at the end:


```
select * from events
order by
CASE WHEN id = 183 THEN '1'
                  WHEN id = 999 THEN '2'
                  WHEN id = 689 THEN '3'
                  WHEN id = 636 THEN '4'
                  ELSE 'z' || name END ASC
```

**Review by motdotla — CHANGES_REQUESTED** [review]

File: app/views/admin/disbursement_new.html.erb [hunk 2]
```diff
@@ -8,13 +8,13 @@
     <p>
       <%= form.label :source_event_id, "Source event" %>
       <br/>
-      <%= form.collection_select(:source_event_id, Event.reorder(:name), :id, :admin_dropdown_description, include_blank: "Select one…") %>
+      <%= form.collection_select(:source_event_id, Event.reorder(CUSTOM_SORT), :id, :admin_dropdown_description, include_blank: "Select one…") %>
```
Comment (motdotla) [inline app/views/admin/disbursement_new.html.erb]: `Event.reorder(Event::CUSTOM_SORT)`

**Review by motdotla — CHANGES_REQUESTED** [review]
couple minor changes and then this looks good

**Comment by garyhtou** [comment]
Yay! I think I got it. Thanks for the help 😄
lmk if any edits should be made

**Review by motdotla — APPROVED** [review]
nice work here

**Pull request merged**